contents/tutorials/ios-feature-flags.md

```diff
@@ -0,0 +1,180 @@
+
+```swift file=ContentView.swift
+import SwiftUI
+
+struct ContentView: View {
+    @State private var navigateToFeatureScreen = false
+
+    var body: some View {
+        NavigationStack {
+            VStack {
+                Image(systemName: "globe")
+                    .imageScale(.large)
+                    .foregroundStyle(.tint)
+                Text("Hello, world!")
+
+                Button("Go to Next Screen") {
+                    navigateToFeatureScreen = true
+                }
+                .padding()
+                .background(Color.blue)
+                .foregroundColor(.white)
+                .cornerRadius(10)
+            }
+            .padding()
+            .navigationDestination(isPresented: $navigateToFeatureScreen) {
+                FeatureScreenView(isFlagEnabled: false) // We'll update this later
+            }
+        }
+    }
+}
+```
+
+Then, create a new SwiftUI View called `FeatureScreenView`. Replace the default code in the new file with the following:
+
+```swift file=FeatureScreenView.swift
+import SwiftUI
+
+struct FeatureScreenView: View {
+    var isFlagEnabled: Bool
+
+    var body: some View {
+        ZStack {
+            Color(isFlagEnabled ? .green : .red)
+                .edgesIgnoringSafeArea(.all)
+            .padding()
+            .cornerRadius(10)
+        }
+    }
+}
+```
+
+Our basic set up is now complete. Build and run your app to see it in action.
+
+![Basic setup of the iOS app](../images/tutorials/ios-feature-flags/basic-app.mp4)
+
+## 2. Add PostHog to your app
+
+With our app set up, it’s time to install and set up PostHog. If you don't have a PostHog instance, you can [sign up for free](https://us.posthog.com/signup).
+
+First, add [posthog-ios](https://github.com/PostHog/posthog-ios) as a dependency to your app using [Swift Package Manager](https://developer.apple.com/documentation/xcode/adding-package-dependencies-to-your-app) (or if you prefer, you can use [CocoaPods](/docs/libraries/ios#cocoapods) or [Carthage](/docs/libraries/ios#carthage)). To add the package dependency to your Xcode project, select `File > Add Package Dependency` and enter the URL `https://github.com/PostHog/posthog-ios.git`. Select `posthog-ios` and click Add Package.
+
+Note that for this tutorial we use version `3.1.3` of the SDK.
+
+![Add PostHog from Swift Package Manager](../images/tutorials/ios-feature-flags/swift-npm.png)
+
+Next, configure your PostHog instance in `App.swift` using your project API key and instance address (you can find these in [your project settings](https://us.posthog.com/project/settings)):
+
+```swift file=App.swift
+import SwiftUI
+import PostHog
+
+@main
+struct App: App {
+    init() {
+        let configuration = PHGPostHogConfiguration(apiKey: "<ph_project_api_key>", host: "<ph_instance_address>")
+        configuration.captureApplicationLifecycleEvents = true
+        configuration.recordScreenViews = true
+        PHGPostHog.setup(with: configuration)
+    }
```

Comment:
Is this right? Maybe there is an update to how we install PostHog, based on [what the docs say](https://posthog.com/docs/libraries/ios)?

I get an error for cannot find `PHGPostHogConfiguration`.

I'm using 3.1.3

Reply:
Yikes. Thought I updated this. Thanks for catching it.

I'll need to update our A/B testing tutorials for iOS and Android too (since they were written with SDK v2, and not V3, so the syntax has changed). Ive made a note to do this